config/locales/en.yml

```diff
@@ -21,3 +21,22 @@
 
 en:
   hello: "Hello world"
+  fullName: "Full name"
```

Comment:
It is better to split translations into scopes (Rails gives a nice way to do this per view/partial). Keeping all translations in the top level can become messy really soon.